src/snowflake/cli/_plugins/connection/util.py

```diff
@@ -62,8 +105,12 @@ def is_regionless_redirect(conn: SnowflakeConnection) -> bool:
     assume it's regionless, as this is true for most production deployments.
     """
     try:
-        *_, cursor = conn.execute_string(REGIONLESS_QUERY, cursor_class=DictCursor)
-        return cursor.fetchone()["REGIONLESS"].lower() == "true"
+        return (
+            get_ui_parameter(
+                conn, UIParameter.ENABLE_REGIONLESS_REDIRECT, "true"
```

Comment:
the previous version didn't assume that `true` was the default, is it safe to change?

Reply:
yeah good point. if this parameter is ever removed from the response, is `true` the default we want to assume?

Reply:
I am not aware of the logic behind it, but I am relying on the default that Cam used. My understanding, this is a configuration rather than a parameter for migration.